compiler-core/src/language_server/engine.rs

```diff
@@ -399,7 +401,13 @@ where
                 if !value.public {
                     continue;
                 }
-                completions.push(value_completion(Some(&alias), name, value));
+
+                let module = match import.as_name.clone() {
+                    Some((AssignName::Variable(name), _)) => Some(name),
+                    Some((AssignName::Discard(_), _)) => None,
+                    None => Some(module.used_name()),
+                };
+                completions.push(value_completion(module.as_deref(), name, value));
```

Comment:
Same as above, this should use `used_name` and should not insert values when the module is discarded.